### DISCUSSION TITLE
Feature request: adjustable splitter in editor

### DISCUSSION THREAD

**Discussion by andersmmg** [bootstrap-email/bootstrap-email#125] [Ideas]
Pretty self-explanitory, this would help a lot with editing and it's not an available feature yet as far as I can tell.

**Comment by stuyam** [comment]
Could you explain what you goal for this is? Is it that you want to adjust the size of the Preview viewport for testing on desktop vs mobile? Do you need different viewport sizes than the Desktop and Mobile options already available. Just looking for the main objective to better help me figure out solutions for this 👍

**Reply by andersmmg** [reply]
I want to adjust it so I can see longer lines in the code editor, as well as adjust the preview size a bit too. The editor is the main thing I want it for though.
